newton/examples/basic/example_basic_reduced_elastic_chair_stick_slip.py

```
@@ -0,0 +1,890 @@
+    cache_root = Path(os.environ.get("NEWTON_CACHE_PATH", Path.home() / ".cache" / "newton"))
+    return cache_root / "polyhaven" / ASSET_NAME
+
+
+def _download_polyhaven_chair() -> Path:
+    asset_dir = _asset_cache_dir()
+    asset_dir.mkdir(parents=True, exist_ok=True)
+    for filename, url in POLYHAVEN_URLS.items():
+        path = asset_dir / filename
+        if path.exists() and path.stat().st_size > 0:
+            continue
+        req = urllib.request.Request(url, headers={"User-Agent": "newton-reduced-elastic-chair-example/1.0"})
```

Ruff (0.16.3)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)